From 2c23cb7b2251551ef31b3521aaf9e2ed8f8acc79 Mon Sep 17 00:00:00 2001
From: Minsoo <gkm19809@gmail.com>
Date: Wed, 17 Aug 2022 05:30:20 +0000
Subject: [PATCH] Use SDL 2.23.2

---
 tools/ports/sdl2.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/ports/sdl2.py b/tools/ports/sdl2.py
index 0114d7934..3b7ce5b65 100644
--- a/tools/ports/sdl2.py
+++ b/tools/ports/sdl2.py
@@ -5,8 +5,8 @@
 
 import os
 
-TAG = '0fcfaf9e9482953ee425cc15b91160b92de3df7f'
-HASH = '2891b65bbe34ada80de6c83751b01cd259f877123f0eeb31cbacf8f7bac06b2157e17a2183f58911a9849ab57ca8aba165d5b3058ecd2d7191bfb61c0595807e'
+TAG = 'cb46e1b3f06a08d57b4ccd83127d1ec3139e1c0f'
+HASH = 'b245b89b81b15233d9db1663ebdfd864ce5dc3481ca974417f5168cc47d5d97861e90f49cfcf7801fe25a5e2a28a45ccf25b79b953e5e556a642eee6bda979a4'
 SUBDIR = 'SDL-' + TAG
 
 
@@ -28,7 +28,7 @@ def get(ports, settings, shared):
     ports.install_headers(source_include_path, target='SDL2')
 
     # build
-    srcs = '''SDL.c SDL_assert.c SDL_dataqueue.c SDL_error.c SDL_guid.c SDL_hints.c SDL_list.c SDL_log.c
+    srcs = '''SDL.c SDL_assert.c SDL_dataqueue.c SDL_error.c SDL_guid.c SDL_hints.c SDL_list.c SDL_log.c SDL_utils.c 
     atomic/SDL_atomic.c atomic/SDL_spinlock.c audio/SDL_audio.c audio/SDL_audiocvt.c audio/SDL_audiodev.c
     audio/SDL_audiotypecvt.c audio/SDL_mixer.c audio/SDL_wave.c cpuinfo/SDL_cpuinfo.c
     dynapi/SDL_dynapi.c events/SDL_clipboardevents.c events/SDL_displayevents.c events/SDL_dropevents.c
-- 
2.37.1